### PR TITLE
Add option for middle click paste

### LINKED ISSUE #23: Add middle-click paste option
According to my [super scientific poll](https://plus.google.com/+CassidyJames/posts/dzjqxfETc51), around 30% of users actively use this feature and and 43% didn't know it existed. But for the 27% who don't use it (or anyone who actively hates it), we can add a toggle in the general mouse/touchpad settings.

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #23 

Should be tested on Loki since this key doesn't/shouldn't exist there

### CODE REVIEW

File: src/Widgets/GeneralSection.vala [hunk 1]
```diff
@@ -54,7 +54,27 @@ public class MouseTouchpad.Widgets.GeneralSection : Gtk.Grid {
         attach (primary_button_switcher, 1, 1, 2, 1);
         attach (new SettingLabel (_("Reveal pointer:")), 0, 2, 1, 1);
         attach (reveal_pointer_switch, 1, 2, 1, 1);
-        attach (help_icon, 2, 2, 1, 1);
+        attach (locate_pointer_help, 2, 2, 1, 1);
+
+        var interface_settings_schema = SettingsSchemaSource.get_default ().lookup ("org.gnome.desktop.interface", false);
+
```
Comment (jeremypw) [inline src/Widgets/GeneralSection.vala]: You need to test whether "null" has been returned before using interface_settings_schema.

I got a crash running this branch on Loki (may or may not be related).
